migration/iD11/oracle/202311051436_IDEMPIERE-5913.sql

```diff
@@ -0,0 +1,17 @@
+SET SQLBLANKLINES ON
+SET DEFINE OFF
+
+-- Nov 5, 2023, 2:36:33 PM WIB
+INSERT INTO AD_SysConfig (AD_SysConfig_ID,AD_Client_ID,AD_Org_ID,Created,Updated,CreatedBy,UpdatedBy,IsActive,Name,Value,Description,EntityType,ConfigurationLevel,AD_SysConfig_UU) VALUES (200240,0,0,TO_TIMESTAMP('2023-11-05 14:36:32','YYYY-MM-DD HH24:MI:SS'),TO_TIMESTAMP('2023-11-05 14:36:32','YYYY-MM-DD HH24:MI:SS'),10,10,'Y','DELETE_IDEMPIERE_LOGS_PAST_DAY','7','delete log files with last modified older than n days of current time','D','S','cbc9ab78-9f21-48df-b516-85003ba01899')
```

Comment:
We don't need a SysConfig here - it sounds better to add a parameter to the process, similar to NoteDelete we can use a parameter with name KeepLogDays